index.html

```diff
@@ -36,6 +36,80 @@ <h2 class="supporting-text">
           <li class="icon"><a href="https://about.meta.com/"><img src="./assets/m.png" alt=""></a></li>
         </ul>
       </section>
-    </div>
-  </body>
-</html
+      <section class="work">
+        <div class="title">
+          <h3>My Recent Works</h3>
+          <hr>
+        </div>
+        <div class="first-div">
+          <img src="./assets/first.png" alt="first">
+          <h4>Multi-Post Stories</h4>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+          <ul class="skills">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="btn-prj">See Porject</button>
+        </div>
+        <div class="project-item">
+          <h5>Profesional Art Printing Data</h5>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+          <ul>
+            <li>HTML</li>
+            <li>Bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <button class="btn-itm">See project</button>
+        </div>
```

Comment:
This section is very well done but there are a few things that you can add to make it even perfect and identical to the Figma design completely. I suggest you check the spacing around these parts of the section as shown in the image below. They are wider than it is in the Figma design 😉 

Your design: 
![image](https://user-images.githubusercontent.com/107795045/227558715-6f51ec83-a888-4f3b-9b94-dbcc26c48e02.png)

Figma design: 
![image](https://user-images.githubusercontent.com/107795045/227558972-5b316ba7-8181-42f3-ab3c-4ccb5324ca7a.png)